jobs/ml-ops/inference/main.py

```diff
@@ -0,0 +1,66 @@
+from fastapi import FastAPI
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import RocCurveDisplay
+import pickle
+import boto3
+import pandas
+import os
+
+import data
+
+classifier = RandomForestClassifier()
+
+app = FastAPI()
+
+MODEL_FILE = "classifier.pkl"
+
+
+class ClassifierLoader(object):
```

Comment:
This is a stylistic difference, but I've always wondered why you inherit from object?

```suggestion
class ClassifierLoader:
```

Reply:
Done